integration/inspektor-gadget/integration_test.go

```diff
@@ -909,6 +911,57 @@ func TestOpensnoop(t *testing.T) {
 	RunCommands(commands, t)
 }
 
+func TestNetworkGraph(t *testing.T) {
+	ns := GenerateTestNamespaceName("test-networkgraph")
+
+	t.Parallel()
+
+	networkGraphCmd := &Command{
+		Name:         "StartNetworkGadget",
+		Cmd:          fmt.Sprintf("$KUBECTL_GADGET trace network -n %s -o json", ns),
+		StartAndStop: true,
+		ExpectedOutputFn: func(output string) error {
+			expectedEntry := &networkTypes.Event{
+				Event: eventtypes.Event{
+					Type: eventtypes.NORMAL,
+					CommonData: eventtypes.CommonData{
+						Namespace: ns,
+						Pod:       "test-pod",
+					},
+				},
+				PktType:    "OUTGOING",
+				Proto:      "tcp",
+				Port:       443,
+				RemoteKind: "other",
+			}
```

Comment:
It should be able to also capture the DNS request to the `svc kube-system/kube-dns`.